src/community/propose-a-content-change-using-github/index.md.njk

```diff
@@ -0,0 +1,93 @@
+---
+title: Propose a content change using GitHub
+description: Find out how to suggest a change to content in the GOV.UK Design System using GitHub
+section: Community
+layout: layout-pane.njk
+order: 2
+---
+
+The GOV.UK Design System team uses a service called GitHub to manage content in the GOV.UK Design System.
+
+This guide explains how to propose a change to the Design System's content. You will need a GitHub account to do this.
+
+If you do not have one already, you can [create a GitHub account for free](https://github.com/).
+
+{% from "inset-text/macro.njk" import govukInsetText %}
+{{ govukInsetText({
+  text: "Rest assured that it's impossible for you to break the Design System by proposing changes. The GOV.UK Design System team reviews all changes before publishing."
+}) }}
+
+If you get stuck whilst following these steps and you need help, you can:
+
+- get in touch on [#govuk-design-system channel on cross-government Slack](https://ukgovernmentdigital.slack.com/app_redirect?channel=govuk-design-system)
+- email the GOV.UK Design System team on <br>govuk-design-system-support@digital.cabinet-office.gov.uk
+
+## 1. Go to the page you want to edit
+
+At the bottom of every page in the Design System you will find a section called "Help improve this page". 
+
+Follow the link to propose a change to the page. This will take you to the page's markdown file. 
+
+You might be told you need to fork the repository to make changes. This is nothing to worry about, it just means making a copy of the Design System that you can edit. Select "fork this repository and propose changes" to continue. 
+
+![](fork-this-repo.png)
+
+## 2. Edit the page file
+
+Edit the markdown to make your change. 
+
+Here is an example showing how to update the description of the checkboxes component.
+
+![The checkboxes description in the file in GitHub with an uppercase C in the word checkboxes](checkboxes-uppercase.png)
+
+In this example, the uppercase ‘C’ on the word ‘Checkbox’ has been changed to lowercase. 
+
+![The checkboxes description in the file in GitHub after the C in the word checkboxes has been changed to lowercase](checkboxes-lowercase.png)
+
+## 3. Propose a change
+
+Once you’re happy, scroll to the bottom of the page until you reach the button that says ‘propose file change’.
+
+Add a short description of what you have changed in the first field. This information will be added to the file’s changelog. Try to be as clear as possible, so that future users can understand what you have done. 
```

Comment:
Suggest adding 'GitHub' to users - to clarify we don't mean end users.

```suggestion
Add a short description of what you've changed in the first field. This information will be added to the file’s changelog. Try to be as clear as possible, so that future GitHub users can understand what you have done. 
```